docs/Reference/TypeScript.md

```diff
@@ -220,18 +220,29 @@ can do it as follows:
 
     const User = Type.Object({
       name: Type.String(),
-      mail: Type.Optional(Type.String({ format: "email" })),
+      mail: Type.Optional(Type.String({ format: 'email' })),
     });
-    type UserType = Static<typeof User>;
+
+    type UserType = Static<typeof User>
     ```
 
 3. Use the defined type and schema during the definition of your route
 
     ```typescript
-    const app = fastify();
+    import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox'
+    import fastify from 'fastify'
+
+    const app = fastify({
+      ajv: {
+        customOptions: {
+          strict: 'log',
+          keywords: ['kind', 'modifier'],
+        },
+      },
+    }).withTypeProvider<TypeBoxTypeProvider>()
```

Comment:
Please check out the example at https://github.com/fastify/fastify-type-provider-typebox. It's different.

Reply:
Thank you, should be updated to match example